nodejs/db2util/README.md

```diff
@@ -0,0 +1,46 @@
+# Node.js SSH Db2 Example
+
+Node.js application that calls Db2 from a cloud-native application, using SSH
```

Comment:
instead of saying "from a cloud-native application", I'd go for something more broad like 
`Node.js application that uses an SSH connection to run Db2 queries. Because no platform-specific modules are required, this technique can be used from any operating system, including Windows, Linux, Mac, IBM i, or even cloud-native applications`